Content.Server/NPC/Pathfinding/PathfindingSystem.cs

```diff
@@ -131,7 +131,6 @@ public override void Update(float frameTime)
                 switch (result)
                 {
                     case PathResult.Continuing:
-                        DebugTools.Assert(path.Frontier.Count > 0);
```

Comment:
Yes this is intentional because I realised when debugging if the path queue times out then the thing never even starts and hence has no frontier.